### PR TITLE
Add fabric monitoring tables definitions.

### PR DESCRIPTION
**Pull request by jfeng-arista**
Add fabric monitoring table names in a common location, as they are utilized in multiple files associated with the fabric monitoring feature.

### CODE REVIEW

**Comment by jfeng-arista** [comment]
The failure was 

2023-07-28T22:55:52.2381925Z ==================================== ERRORS ====================================
2023-07-28T22:55:52.2382365Z ___________ ERROR at setup of TestAcl.test_AclTableCreationDeletion ____________
2023-07-28T22:55:52.2382548Z 
2023-07-28T22:55:52.2383106Z tp = <class 'AssertionError'>, value = None, tb = None
2023-07-28T22:55:52.2383225Z 
2023-07-28T22:55:52.2387164Z     def reraise(tp, value, tb=None):
2023-07-28T22:55:52.2387402Z         try:
2023-07-28T22:55:52.2387618Z             if value is None:
2023-07-28T22:55:52.2387832Z                 value = tp()
2023-07-28T22:55:52.2388083Z             if value.__traceback__ is not tb:
2023-07-28T22:55:52.2388615Z >               raise value.with_traceback(tb)

File: common/schema.h [hunk 1]
```diff
@@ -530,6 +530,11 @@ namespace swss {
 #define CFG_DTEL_QUEUE_REPORT_TABLE_NAME	"DTEL_QUEUE_REPORT"
 #define CFG_DTEL_EVENT_TABLE_NAME			"DTEL_EVENT"
 
+#define CFG_FABRIC_MONITOR_DATA_TABLE_NAME	"FABRIC_MONITOR"
+#define CFG_FABRIC_MONITOR_PORT_TABLE_NAME	"FABRIC_PORT"
+#define APP_FABRIC_MONITOR_DATA_TABLE_NAME	"FABRIC_MONITOR"
```
Comment (qiluo-msft) [inline common/schema.h:535]: I see the same name in both ConfigDB and ApplDB. I am not sure if orchagent could tell the difference if you subscribe both. If you have this use case, you may need to rename one of them.

Comment (jfeng-arista) [reply common/schema.h:535]: Based on my understanding,  the current way of orchagents react to the confg change is that: cfgmgr reacts the change in config_db and write to appl_db, and orchagent only reacts to the appl_db. So the the fabricportsorch only subscribe the one in appl_db. Let me know if it is ok in this case.  Thanks

**Comment by rlhui** [comment]
Is this a code refactoring only or it's to actually fix any bug?  Thanks.

**Comment by jfeng-arista** [comment]
> 

These are the table defination for fabric link monitoring feature, the other code will depend on these tables

**Comment by jfeng-arista** [comment]
it failed with

In file included from dash/dashaclorch.h:19,
                 from orchdaemon.h:49,
                 from main.cpp:25:
dash/dashorch.h:21:10: fatal error: dash_api/appliance.pb.h: No such file or directory
   21 | #include "dash_api/appliance.pb.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

It seems not related to this change

**Comment by jfeng-arista** [comment]
test_empty_trap_cfg failed (1 runs remaining out of 2).
	<class 'AssertionError'>
	assert True == False
  -True
  +False
	[<TracebackEntry /agent/_work/1/s/sonic-swss/tests/test_copp.py:771>]
test_empty_trap_cfg failed; it passed 0 out of the required 1 times.
	<class 'AssertionError'>
	assert True == False
  -True
  +False
	[<TracebackEntry /agent/_work/1/s/sonic-swss/tests/test_copp.py:771>]
test_disabled_feature_always_enabled_trap failed (1 runs remaining out of 2).
	<class 'AssertionError'>
	assert True == False
  -True
  +False
	[<TracebackEntry /agent/_work/1/s/sonic-swss/tests/test_copp.py:843>]
	
	do not look like related to this change

**Review by arlakshm — APPROVED** [review]

**Comment by arlakshm** [comment]
@qiluo-msft, can you help signoff this PR

**Comment by arlakshm** [comment]
@jfeng-arista, can you help update the branch

**Comment by jfeng-arista** [comment]
> @jfeng-arista, can you help update the branch

done

File: common/schema.h (hunk 1 above)
Comment (arlakshm) [reply common/schema.h:536]: Hi @jfeng-arista, I think @qiluo-msft concern is correct. I think the table names cannot be same. In orchagent,  in doTask we do `table_name = consumer.getTableName();` , which is getting from `m_tableName` So if APP_DB and CONFIG_DB have the same table name, orchagent does not  know which db the entry belongs to. Please check

Comment (jfeng-arista) [reply common/schema.h:536]: I can change the name if you prefer. 
But I have the code internally working with these,  where we set the config_db via cli, and get the notification from the config_db and cfgmgr copy the change in config_db to appl_db.   orchagent/fabricportsorch ONLY subscribes appl_db and reacts to its change, orchagent/fabricportsorch does not subscribe / react to config_db change.

**Review by qiluo-msft — APPROVED** [review]

**Comment by jfeng-arista** [comment]
test_NeighborAddRemoveIpv6LinkLocal failed (1 runs remaining out of 2).
	<class 'AssertionError'>
	assert 3 == 4
  -3
  +4
it looks like not related to this change

**Comment by jfeng-arista** [comment]
test_NeighborAddRemoveIpv6LinkLocal failed (1 runs remaining out of 2).
	<class 'AssertionError'>
	assert 3 == 4
  -3
  +4
	[<TracebackEntry /agent/_work/1/s/sonic-swss/tests/test_ipv6_link_local.py:63>]

The failure not related to this change

**Review by arlakshm — APPROVED** [review]

**Pull request merged**